README.md

```diff
@@ -25,3 +25,11 @@ The person that takes attendance, notes, and assigmments during the meeting and
 ![image](https://user-images.githubusercontent.com/9609562/220431948-2b362022-7d6f-47d5-b211-6a1811509ada.png)
 
 Your roles documentation site should be live at https://innersourcecommons.github.io/your-working-group/!
+
+## Contributing
+
+* Prefer asterisk formatting of lists rather than hyphen formatting.
+See [this discussion](https://nickang.com/2018-03-05-markdown-bullet-points-hyphen-asterisk/).
+* Write each new sentence on a new line. GitHub allows leaving comments on a line-by-line basis.
+Review and comment on the content of submitted text is much easier if there are multiple lines on-which to leave comments.
```

Comment:
```suggestion
Reviewing and commenting the submitted text is much easier if there are multiple lines on which to leave comments.
```

The "on the content" did not seem to add extra value.

Also looked up "on-which" in the [dictionary](https://www.merriam-webster.com/dictionary/on-which) and could not find it :)